DS-203 Add className wrapper to Spinner API

gregaubert opened the pull request.

david-cho-lerat-sonarsource commented on src/components/spinner/Spinner.tsx:
```diff
@@ -20,26 +20,34 @@
 
 interface Props {
   className?: string;
+  classNameWrapper?: string;
```
using `wrapperClassName` would be less confusing and more accurate IMO

gregaubert replied:
Ok I'll change it!

david-cho-lerat-sonarsource reviewed (changes requested):
This isn't really adding a "large spinner", but rather a "stylable spinner", right? You can use that custom style to make it large, or anything else

gregaubert commented:
> This isn't really adding a "large spinner", but rather a "stylable spinner", right? You can use that custom style to make it large, or anything else

Yeah indeed, at first I added a isLarge prop and then I noticed there were other use cases and changed the implem and forgot to rename the PR.

david-cho-lerat-sonarsource reviewed (approved):
LGTM, thanks! :+1:
